test/.eslintrc.yaml

```diff
@@ -19,6 +19,33 @@ rules:
   ## common module is mandatory in tests
   node-core/required-modules: [error, common]
 
+  no-restricted-syntax:
+    # Config copied from .eslintrc.js
+    - error
+    - selector: "CallExpression[callee.object.name='assert'][callee.property.name='doesNotThrow']"
+      message: "Please replace `assert.doesNotThrow()` and add a comment next to the code instead."
```

Comment:
Is this doesNotThrow stuff a separate and unrelated change?

Reply:
Never mind. I misunderstood the PR text. Deleting previous comment. Ignore. Thanks! :-D